tests/modelfree/test_experiments.py

```diff
@@ -86,10 +96,16 @@ def test_score_agent(config):
     {
         'env_name': 'multicomp/SumoHumansAutoContact-v0',
         'adv_noise_params': {'noise_val': 0.1},
-    }
+    },
+    {
+        'rl_algo': 'gail',
+        'num_env': 1,
+        'expert_dataset_path': 'tests/modelfree/SumoAnts_traj/agent_0.npz',
```

Comment:
Best to avoid large binary files in repos as it slows down cloning. It's worth it for a test case, but can we maybe make the dataset file smaller? 3 MB is quite a lot (would expect to correspond to many episodes). Also if we only use the `agent_0` one, can we remove the `agent_1` file?

Reply:
each file corresponds to only 20 episodes actually. Recall that a file with only 5000 episodes was taking > 1 GB of data in S3. Nonetheless i'll make it like 5 episodes instead lol

Reply:
:+1: for a test even just two episodes is enough. (1 maybe not, since there could be edge cases where it fails with >1 episode.)